### PR TITLE
add papirus-icon-theme

### PR DESCRIPTION
**Pull request by Yisus7u7**
Papirus is a very popular icon theme available in all Linux distros

### CODE REVIEW

File: packages/papirus-icon-theme/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/PapirusDevelopmentTeam/papirus-icon-theme
+TERMUX_PKG_DESCRIPTION="Papirus open source icon theme for Linux
+ Papirus is a SVG-based icon theme, drawing inspiration from Material Design
+ and flat design.
+ .
+ This package contains the following icon themes:
```
Comment (suhan-paradkar) [inline packages/papirus-icon-theme/build.sh:6]: Please Keep the description in a single line..
Apt doesn't support multiline description very well

File: packages/papirus-icon-theme/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - Papirus
+  - Papirus-Dark
+  - Papirus-Light
+  - ePapirus"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=20210601
+TERMUX_PKG_SRCURL=https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SHA256=e39f4212e7433010b6246f5a2989a9e263c7ab8029a5d1759f2744f4a54b6b14
+TERMUX_PKG_DEPENDS="hicolor-icon-theme"
```
Comment (suhan-paradkar) [inline packages/papirus-icon-theme/build.sh:16]: You need to add relevant commands to execute below under the function termux_step_pre_configure or termux_step_configure and other functions... Please check the Contributor's wiki for relevant information

**Review by suhan-paradkar — CHANGES_REQUESTED** [review]

File: packages/papirus-icon-theme/build.sh (hunk 2 above)
Comment (Yisus7u7) [reply packages/papirus-icon-theme/build.sh:16]: I do not now what to do and much less with this message from the Makefile:

```
make: Nothing to be done for 'all'.
```

**Review by xeffyr — CHANGES_REQUESTED** [review]
1. Remove multiline description. Also ensure it is short but well defined.
2. Don't think that ./build-package.sh will do everything for you automatically. Sometimes you will need to put additional options or override default configure or build steps. Look the package source code and then make decision how build.sh should like. Since you're submitting a pull request, maintainers won't do that for you.

File: packages/papirus-icon-theme/build.sh (hunk 2 above)
Comment (xeffyr) [reply packages/papirus-icon-theme/build.sh:16]: > make: Nothing to be done for 'all'.

Does package uses a build system supporting out-of-tree builds? If no \-\-\> use `TERMUX_PKG_BUILD_IN_SRC=true`.

Comment (Yisus7u7) [reply packages/papirus-icon-theme/build.sh:16]: Ok

**Comment by Yisus7u7** [comment]
I think it would be easier if I could run the build system to test right away, but unfortunately I don't have space to install all those things

File: packages/papirus-icon-theme/build.sh (hunk 2 above)
Comment (Yisus7u7) [reply packages/papirus-icon-theme/build.sh]: It seems that it is already solved : 

```
/build-package.sh papirus-icon-themeBuilding dependency libc++ if necessary...
libc++@21d-1 installed - skipping
Building dependency hicolor-icon-theme if necessary...
hicolor-icon-theme@0.17-26 installed - skipping
termux - building papirus-icon-theme for arch arm...
Downloading https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/archive/refs/tags/20210601.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:100   161  100   161    0     0    347      0 --:--:-- --:--:-- --:--:--   348
100  122k    0  122k    0     0  99197      0 --:--:--  0:00:01 --:--:100 3685k    0 3685k    0     0  1625k      0 --:--:--  0:00:02 --:--:100 7455k    0 7455k    0     0  2281k      0 --:--:--  0:00:03 --:--:100 10.3M    0 10.3M    0     0  2360k      0 --:--:--  0:00:04 --:--:100 14.6M    0 14.6M    0     0  2845k      0 --:--:--  0:00:05 --:--:100 18.3M    0 18.3M    0     0  2993k      0 --:--:--  0:00:06 --:--:100 19.1M    0 19.1M    0     0  2470k      0 --:--:--  0:00:07 --:--:100 21.9M    0 21.9M    0     0  2716k      0 --:--:--  0:00:08 --:--:100 25.6M    0 25.6M    0     0  2830k      0 --:--:--  0:00:09 --:--:100 26.5M    0 26.5M    0     0  2840k      0 --:--:--  0:00:09 --:--:-- 2834k
make: Nothing to be done for 'all'.
mkdir -p /data/data/com.termux/files/usr/share/icons
cp -R ./Papirus-Dark ./Papirus-Light ./Papirus ./ePapirus /data/data/com.termux/files/usr/share/icons
```

Comment (Yisus7u7) [reply packages/papirus-icon-theme/build.sh]: Pd: It's an icon issue, I forgot to specify that the architecture was all

**Comment by Yisus7u7** [comment]
Pd: The architecture must be "all"

**Comment by Yisus7u7** [comment]
It's my first time with this, and I look forward to collaborating more in the future.

**Comment by Yisus7u7** [comment]
@xeffyr If this is an icon pack, why is it being compiled to multiple architectures?, or is that just something github actions doing?

**Review by xeffyr — APPROVED** [review]

**Comment by WMCB-Tech** [comment]
> @xeffyr If this is an icon pack, why is it being compiled to multiple architectures?

@Yisus7u7 If a package is a documentation files, manpage extras, gtk/icon themes, you could specify:
```
TERMUX_PKG_PLATFORM_INDEPENDENT=true
```

In your package `build.sh` script to tell that the package is for `all` architectures

**Pull request merged**